source/uwp/Renderer/lib/Util.cpp

```diff
@@ -481,25 +481,28 @@ bool MeetsRequirements(winrt::IAdaptiveCardElement const& cardElement, winrt::Ad
     winrt::IVector<winrt::AdaptiveRequirement> requirements = cardElement.Requirements();
     bool meetsRequirementsLocal = true;
 
-    for (auto req : requirements)
+    if (requirements)
```

Comment:
what is the reason for this variable to be null?

Reply:
@beervoley I'm not sure. We had a null check before switching to WinRT, and I added it back.

Without it, we crash at one of our test cases.